### DISCUSSION TITLE
adding voice recorder

### DISCUSSION THREAD

**Discussion by View-my-Git-Lab-krafi** [TriliumNext/Trilium#4489] [Ideas]
well i made a python application that can encrypt photo , voice recorder from inbuilt voice record or external file , also sync with git.

**now basically i want to add voice recording feature to trillium ...**

https://gitlab.com/krafi/git-backed-diary

**Comment by View-my-Git-Lab-krafi** [comment]
i also want to to how to use encrypting feature on trilium

**Reply by zadam** [reply]
Protected notes are documented here: https://github.com/zadam/trilium/wiki/Protected-notes
